### ISSUE TITLE
Use learning rate finder to determine optimal learning rate

### ISSUE THREAD

**Issue by dufaultc** [McMasterAI/Radiology-and-AI#50]
-Use learning_rate_finding notebook to see which learning rate gives lowest learning rate after 2 epochs
-The notebook is under the training-improvments-47 branch
-The only changes you should have to make are in sections 3 and 4 of the notebook, and in the first cell change 
"!git clone https://github.com/McMasterAI/Radiology-and-AI.git" to "!git clone --branch training-improvements-47 https://github.com/McMasterAI/Radiology-and-AI.git"
-Find which accumulate_grad_batches amount finds best learning rate (maybe try 10, 15 and 20)
-Check 50 different learning rates for each 
-Each should give different optimal learning rate
-Of these three optimal learning rates, run model training with each and see which gives lowest loss value after 2 epochs of training
-You will need to create a wandb account to track the model training and also make a shortcut to the macai datasets folder I shared with you in your base drive folder
-You don't need to push any changes to the repo just report the best learning rate you found and anything else you think important
